extensions/amp-ad/validator-amp-ad.protoascii

```diff
@@ -64,6 +64,7 @@ tags: {  # <amp-ad>
   }
   attrs: { name: "template" }
   attrs: { name: "type" mandatory: true }
+  attrs: { name: "sticky" }
```

Comment:
Does this accept values? Your example doesn't suggest that it does. If not, you may want `value: ""` to indicate that no values are allowed.

Reply:
Yes, you are right. Thanks for the suggestion!